scripts/generate-rtk-apis.ts

```diff
@@ -72,6 +72,7 @@ const config: ConfigFile = {
       filterEndpoints: ['listPlaylist', 'getPlaylist', 'createPlaylist', 'deletePlaylist', 'replacePlaylist'],
       tag: true,
     },
+    // PLOP_INJECT_API_CLIENT
```

Comment:
can we have a comment here explaining that this is used by the api client generator as well 🙏